### ISSUE TITLE
Replacement for Socicon

### ISSUE THREAD

**Issue by brendo** [squidfunk/mkdocs-material#79]
Thoughts on using [Font Awesome](http://fontawesome.io/icons/) to replace Socicon?

FA has a wider variety of icons that are available (eg. Slack) and has massive community support, with over a million raised on [Kickstarter](https://www.kickstarter.com/projects/232193852/font-awesome-5) for the next version.

I feel like FA allows writers to customise the footer to contain information that might not be considered as 'social', such as a deep link to an AWS login, a link to metric and graphs relevant to these docs etc.

**Comment by squidfunk** [comment]
FontAwesome doesn't include all the social icons, I think Socion has far better coverage. You're right in general considering the support of FA, but I don't know... Material icons AND Font Awesome?

If the user needs more configuration he can override separate parts of the theme, so I would leave the social option as a default for now. Let's see how it will be used.

**Comment by brendo** [comment]
That's fair, there's nothing stopping a user using `extra_css` and adding FA to the site.

I'm not sure I can agree with Socion's coverage, there is a considerable number of [brand icons](http://fontawesome.io/icons/#brand) available from FA. Though I can spot some differences, particularly with gaming related icons 👍

**Comment by squidfunk** [comment]
Okay, I checked Font Awesome and you're right, they are a lot of brand icons. Therefore I decided to give Font Awesome a try, as it's far more wide spread- Furthermore I put the social links inside an own partial called `social.html` and wrapped them inside a block called `social`, so when a replacement is desired, it should be pretty straight forward. I think that should cover pretty much every case. Also adjusted the documentation.
